libpod/runtime_pod_linux.go

```diff
@@ -328,7 +315,16 @@ func (r *Runtime) removePod(ctx context.Context, p *Pod, removeCtrs, force bool,
 				}
 			}
 			if err == nil {
-				if err := conmonCgroup.Delete(); err != nil {
+				for attempts := 0; attempts < 50; attempts++ {
+					err = conmonCgroup.Delete()
+					if err == nil || os.IsNotExist(err) {
+						// success
+						err = nil
+						break
+					}
+					time.Sleep(time.Millisecond * 100)
+				}
+				if err != nil {
```

Comment:
Any use to having at least a logdebug here if attempts are >= 50 just stating that we ran out of attempts?